backend/src/controllers/users.js

```
@@ -45,14 +34,24 @@ export async function insertUsers(req, res) {
 
         if (findexistinguser.length > 0) {
-            return res.status(200).json({ user: {findexistinguser}, message: "User already exists" });
+            return res.status(200).json({ user: { findexistinguser }, message: "User already exists" });
         }
 
-        const { clerk_id, email, firstName, lastName, username, number, img } = req.body
+        const clerkUser = await clerkClient.users.getUser(userId);
+
+        const userData = {
+            clerk_id: userId,
+            username: clerkUser.username || null,
```

Biome (1.9.4)

[error] This property value named username is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named username


(lint/suspicious/noDuplicateObjectKeys)